spectator-api/src/main/java/com/netflix/spectator/impl/UnsafeUtils.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014-2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spectator.impl;
+
+import sun.misc.Unsafe; // NOPMD
```

Comment:
wow, guess it's still good through JDK 20

Reply:
Yeah, there are some other options, but are kind of a pain until we can drop java 8 support.